### ISSUE TITLE
Implement "Go to definition" functionality in AXAML code

### ISSUE THREAD

**Issue by nicola36631** [AvaloniaUI/Avalonia#12761]
# **Is your feature request related to a problem? Please describe.**
While developing an Avalonia application I am constantly dearly missing the "Go to definition" functionality in AXAML code. Having to manually search for properties, types, resource dictionaries, templates and everything else adds up very quickly.

# **Describe the solution you'd like**
"Go to definition" should be implemented and it should automatically bring you to the definition of the element under the caret (be it in C# code or in AXAML code).

# **Describe alternatives you've considered**
None.

# **Additional context**
Labels : [area-dev-tools] [enhancement]

**Comment by Mrxx99** [comment]
Shouldn't this be an issue in https://github.com/AvaloniaUI/AvaloniaVS ?

Maybe it can be moved by the avalonia team

**Comment by MrJul** [comment]
Indeed, moving to AvaloniaVS (Rider has built-in *Go to definition* support for Avalonia).
